openid-connect-server/src/test/java/org/mitre/oauth2/service/impl/TestDefaultSystemScopeService.java

```diff
@@ -184,6 +184,60 @@ public void scopesMatch() {
 
 		// extra scope (fail)
 		assertThat(service.scopesMatch(expected, actualBad), is(false));
+		
 	}
+	
+	@Test
+	public void getremoveRestrictedAndReservedScopes() {
+						
+		Set<SystemScope> restrictedAndReservedScopes = Sets.newHashSet(defaultDynScope1, defaultDynScope2, dynScope1);
+		assertThat(service.removeRestrictedAndReservedScopes(allScopes),equalTo(restrictedAndReservedScopes));
+	}
+	
+	@Test
+	public void getremoveReservedScopes() {
+		Set<SystemScope> removeReservedScopes = Sets.newHashSet(defaultDynScope2, defaultDynScope1, defaultScope2,restrictedScope1,defaultScope1,dynScope1);
+		assertThat(service.removeReservedScopes(allScopes), equalTo(removeReservedScopes));
+	}
+	
+	@Test 
+	public void getById() {
+		
+		// check null condition
+
+		assertThat(service.getById((long) 60), is(nullValue()));
+
+		
+	}
+	
+	@Test 
+	public void getByValue() {
+		
+		// check null condition
+		
+		assertThat(service.getByValue("defaultDynScope1String"), is(nullValue()));		
+		
+		
+	}
+	
+	@Test
+	public void save() {
+		
+		// check null condition 
+
+		assertThat(service.save(defaultScope1), is(nullValue()));
+
+
+		
+	}
+	
+	@Test
+	public void getReserved() {
+		//Set<SystemScope> restricted = Sets.newHashSet(defaultScope1, defaultScope2, restrictedScope1);
+		Set<SystemScope> reserved = service.getReserved();
```

Comment:
This is a pointless test because it's comparing a value to itself without making any changes in between. Better to copy or replicate the reserved scopes here so that if the reserved set changes in the future the unit tests will alert someone to that. Alternatively, you can pull the "reserved" set directly from the SystemScopeService interface instead of the instance object here, and then compare that to the instance below. Either way, calling the same function twice and getting the same result is a non-test.